### DISCUSSION TITLE
Added sucesfully but no data?

### DISCUSSION THREAD

**Discussion by richardcavspain** [wills106/homeassistant-solax-modbus#185] [General]
Hi, apologies if this is the wrong place to post, I am very new to HA and HACS but following the instructions and a LOT of Googling, I have managed to sucessfully add the homeassistant-solax-modbus to my HA setup through HACS - in fact this is the ONLY reason I am using HA if I'm honest.

I added the IP address and port and its reporting as being sucessfully added, I rebooted and it reconnected on reboot.

The Solax API reports as "OK" but even after waiting over an hour (in case of a delay) I see only 0W next to all of the interesting items.
Could anyone point me in the right direction as to figure out what I have done wrong please?

Regards

Richard

**Comment by wills106** [comment]
Could you raise an issue as a bug, there is a [template for you to follow](https://github.com/wills106/homeassistant-solax-modbus/issues/new/choose)
It will be easier once you have filled in the info for us to troubleshoot.

**Comment by richardcavspain** [comment]
Thanks, I have done so now, I think the issue might be related to a missing serial number in the config after looking at the logs?
